src/Akka.Persistence.Azure.Tests/Hosting/AzurePersistenceHostingSanityCheck.cs

```diff
@@ -20,6 +21,7 @@ public class AzurePersistenceHostingSanityCheck
         public static async Task<IHost> StartHost(Action<AkkaConfigurationBuilder> testSetup)
         {
             var conn = Environment.GetEnvironmentVariable("AZURE_CONNECTION_STR") ?? "UseDevelopmentStorage=true";
+            await DbUtils.CleanupCloudTable(conn);
```

Comment:
Can we add a separate test to this file that uses the `DefaultAzureCredential` ?

Reply:
We're hitting the same problem as Akka.Discovery.Azure here, `DefaultAzureCredential` needs HTTPS and bearer token to work, which Azurite does not support

Reply:
Ok, got it.